frontend/packages/topology/src/components/DefaultCreateConnector.tsx

```diff
@@ -23,7 +23,7 @@ const DefaultCreateConnector: React.FC<DefaultCreateConnectorProps> = ({
       x2={endPoint.x}
       y2={endPoint.y}
     />
-    {hints && hints.length === 1 && hints[0] === 'create' ? (
+    {hints && hints[hints.length - 1] === 'create' ? (
```

Comment:
modified the check to allow the latest hint to decide the connector type
cc @christianvogt